packages/mock-server/src/data/data.ts

```diff
@@ -234,11 +252,12 @@ const createStatusLog = (status: string, entered: Date) => {
   return statusTimes;
 };
 
+const shippingMethods = ['Air', 'Sea', 'Road', 'Rail'];
```

Comment:
🤷

Reply:
ditto.. 🤷  though these are sensible to me